changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.11.0~rc5
+
+* Cleanup sshd config (#3979)
```

Comment:
We should say explicitly in the changelog that this is a security-related bugfix "Security bugfix: Disable unnecessary SSH config options". 

What do you think @emkll?